packages/react-spinner/src/components/Spinner/Spinner.test.tsx

```diff
@@ -19,10 +19,25 @@ describe('Spinner', () => {
     disabledTests: ['component-has-static-classname', 'component-has-static-classname-exported'],
   });
 
-  // TODO add more tests here, and create visual regression tests in /apps/vr-tests
-
   it('renders a default state', () => {
     const result = render(<Spinner>Default Spinner</Spinner>);
     expect(result.container).toMatchSnapshot();
   });
+
+  it('renders Spinner with a label', () => {
+    const result = render(<Spinner label="Loading" />);
+    expect(result.container).toMatchSnapshot();
+  });
+
+  it('doesnt render svg when status is inactive', () => {
+    const result = render(<Spinner status="inactive" />);
+    expect(result.queryByRole('progessbar')).toBeNull();
+  });
+
+  describe('Accessibility Tests', () => {
```

Comment:
For the accessibility tests, you might want to add an issue and do this: https://github.com/microsoft/fluentui/wiki/Manual-Accessibility-Review-Checklist